### ISSUE TITLE
One-way tracing 

### ISSUE THREAD

**Issue by gingercookiemage** [Yelp/py_zipkin#151]
Hi, 
I am trying to enable tracing over rabbitmq messages. We have a few services that communicate over http and also send/receive events to rmq channels which then start further actions. The goal is to connect these requests and messages in the unified trace to better see causality behind certain actions.
My approach was to start a new span in publish_message / recieve_message methods and store/read zipkin headers from message headers. Example span when publishing: 

```
with zipkin_span( service_name='some name', span_name='some name', kind=Kind.PRODUCER ):
     headers = create_http_headers_for_new_span()

```
Receiving message is similar but there I put kind=Kind.CONSUMER.
I have a problem with kind option here is the exact error:

`ERROR: Error emitting zipkin trace. ZipkinError("(logging_helper.py:110) KeyError: <Kind.PRODUCER: 'PRODUCER'>") [py_zipkin.zipkin:500]`

don't understand why this is happening, are producer/consumer spans supported in version 0.20.0 ?

**Comment by drolando** [comment]
We don't use them internally, so there might be some bug in the code regarding Kind.PRODUCER and CONSUMER. Let me have a look

**Comment by drolando** [comment]
@gingercookiemage what encoding are you using? V2 json? protobuf?

**Comment by gingercookiemage** [comment]
@drolando In debug mode I see that encoder is set to `<py_zipkin.encoding._encoders._V1ThriftEncoder>` , and the exception(error message from previous post) is thrown here: `self.logging_context.stop() line 497 of zipkin.py`

**Comment by gingercookiemage** [comment]
Hmmm seems like the problem is here  py_zipkin/encoding/_helpers.py 
```
if self.kind != Kind.LOCAL:
            # If kind is not LOCAL, then we only want client or
            # server side annotations.
            for ann in _DROP_ANNOTATIONS_BY_KIND[self.kind]:
                del full_annotations[ann]

```
```
_DROP_ANNOTATIONS_BY_KIND = {
    Kind.CLIENT: {"ss", "sr"},
    Kind.SERVER: {"cs", "cr"},
}
```
here the Kind.PRODUCER and Kind.CONSUMER should probably  be added.

**Comment by gingercookiemage** [comment]
Hey, do we have any info about this issue?

**Comment by drolando** [comment]
Sorry I implemented support in #153 but forgot to merge it. It should be available in a bit as version `0.20.1`. Let me know if that  works fine for you.

**Comment by gingercookiemage** [comment]
Thanks @drolando! :)
